### PR TITLE
Add What's new in September

### PR DESCRIPTION
**Pull request by BillWagner**
Run the automated tool for September 2020.

[Internal preview link](https://review.docs.microsoft.com/en-us/dotnet/whats-new/2020-09?branch=pr-en-us-20890)

### CODE REVIEW

File: docs/whats-new/2020-09.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+- [NextTurn](https://github.com/NextTurn) - Next Turn (22)
+- [nschonni](https://github.com/nschonni) - Nick Schonning (22)
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (22)
+- [zakaria-c](https://github.com/zakaria-c) - Zakaria (11)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (7)
+- [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (6)
+- [JohyPark](https://github.com/JohyPark) - Joohyung Park (5)
+- [xTyler73](https://github.com/xTyler73) (5)
+- [DCtheGeek](https://github.com/DCtheGeek) - David Coulter (3)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:94]:
```suggestion
```

File: docs/whats-new/2020-09.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [NextTurn](https://github.com/NextTurn) - Next Turn (22)
+- [nschonni](https://github.com/nschonni) - Nick Schonning (22)
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (22)
+- [zakaria-c](https://github.com/zakaria-c) - Zakaria (11)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (7)
+- [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (6)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:91]:
```suggestion
```

File: docs/whats-new/2020-09.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+- [mialkin](https://github.com/mialkin) - Aleksei Mialkin (1)
+- [MisterSadness](https://github.com/MisterSadness) - Piotr Lewandowski (1)
+- [mohsinnasir](https://github.com/mohsinnasir) - Mohsin Nasir (1)
+- [mr5z](https://github.com/mr5z) - mark (1)
+- [n1c](https://github.com/n1c) - Nic (1)
+- [Naine](https://github.com/Naine) - Nathan Williams (1)
+- [netchkin](https://github.com/netchkin) - Pavel Janečka (1)
+- [patgoull](https://github.com/patgoull) (1)
+- [perlun](https://github.com/perlun) - Per Lundberg (1)
+- [robvet](https://github.com/robvet) - Rob Vettor (1)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:125]:
```suggestion
```

File: docs/whats-new/2020-09.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+- [bnewcomb](https://github.com/bnewcomb) - Ben Newcomb (2)
+- [Marusyk](https://github.com/Marusyk) - Roman Marusyk (2)
+- [mohsin-mehmood](https://github.com/mohsin-mehmood) - Mohsin Mehmood (2)
+- [Alexzander1990](https://github.com/Alexzander1990) (1)
+- [aymanizz](https://github.com/aymanizz) - Ayman Izzeldin (1)
+- [bklebe](https://github.com/bklebe) - Beatrix Klebe (1)
+- [bradriske](https://github.com/bradriske) (1)
+- [brendan-mcmahon](https://github.com/brendan-mcmahon) - Brendan McMahon (1)
+- [CAS0](https://github.com/CAS0) - Tony Caso (1)
+- [damabe](https://github.com/damabe) - Dan Mabee (1)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:104]:
```suggestion
```

**Review by scottaddie — CHANGES_REQUESTED** [review]

**Comment by BillWagner** [comment]
Thanks @scottaddie I removed the FTEs and vendors that made weren't identified.

File: docs/whats-new/2020-09.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+- [brendan-mcmahon](https://github.com/brendan-mcmahon) - Brendan McMahon (1)
+- [CAS0](https://github.com/CAS0) - Tony Caso (1)
+- [felgar73](https://github.com/felgar73) - Felix Garcia (1)
+- [gurmeetsinghdke](https://github.com/gurmeetsinghdke) - Gurmeet Singh (1)
+- [hiMinju](https://github.com/hiMinju) - minju (1)
+- [iAliJ](https://github.com/iAliJ) - Ali (1)
+- [InstanceFactory](https://github.com/InstanceFactory) (1)
+- [jmia](https://github.com/jmia) - Jen Armstrong (1)
+- [kotx](https://github.com/kotx) - Kot (1)
+- [kyleburnsdev](https://github.com/kyleburnsdev) - Kyle Burns (1)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:109]:
```suggestion
```

File: docs/whats-new/2020-09.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+- [lbargaoanu](https://github.com/lbargaoanu) - Lucian Bargaoanu (1)
+- [marktab](https://github.com/marktab) - MarkTab marktab.net (1)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:111]:
```suggestion
```

File: docs/whats-new/2020-09.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+- [mohsinnasir](https://github.com/mohsinnasir) - Mohsin Nasir (1)
+- [mr5z](https://github.com/mr5z) - mark (1)
+- [n1c](https://github.com/n1c) - Nic (1)
+- [Naine](https://github.com/Naine) - Nathan Williams (1)
+- [netchkin](https://github.com/netchkin) - Pavel Janečka (1)
+- [patgoull](https://github.com/patgoull) (1)
+- [perlun](https://github.com/perlun) - Per Lundberg (1)
+- [tama](https://github.com/tama) (1)
+- [ThomasArdal](https://github.com/ThomasArdal) - Thomas Ardal (1)
+- [Varad25](https://github.com/Varad25) - Varad Chemburkar (1)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:124]:
```suggestion
```

File: docs/whats-new/2020-09.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+- [bnewcomb](https://github.com/bnewcomb) - Ben Newcomb (2)
+- [Marusyk](https://github.com/Marusyk) - Roman Marusyk (2)
+- [mohsin-mehmood](https://github.com/mohsin-mehmood) - Mohsin Mehmood (2)
+- [Alexzander1990](https://github.com/Alexzander1990) (1)
+- [aymanizz](https://github.com/aymanizz) - Ayman Izzeldin (1)
+- [bklebe](https://github.com/bklebe) - Beatrix Klebe (1)
+- [bradriske](https://github.com/bradriske) (1)
+- [brendan-mcmahon](https://github.com/brendan-mcmahon) - Brendan McMahon (1)
+- [CAS0](https://github.com/CAS0) - Tony Caso (1)
+- [felgar73](https://github.com/felgar73) - Felix Garcia (1)
```
Comment (scottaddie) [inline docs/whats-new/2020-09.md:102]:
```suggestion
```

**Review by scottaddie — CHANGES_REQUESTED** [review]

**Review by scottaddie — APPROVED** [review]
:shipit:

File: docs/whats-new/2020-09.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+### Updated articles
+
+- [Tutorial: Get started with .NET for Apache Spark](../spark/tutorials/get-started.md) - Spark | Update getting started tutorial
+
+## .NET fundamentals
+
+### New articles
+
+- [Configuration files for code analysis rules](../fundamentals/code-analysis/configuration-files.md) - Add documentation for Global AnalyzerConfig
```
Comment (gewarren) [inline docs/whats-new/2020-09.md:52]: I'm not sure why the code quality rules work doesn't show up here.

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/
https://github.com/dotnet/docs/pull/20702

Comment (BillWagner) [reply docs/whats-new/2020-09.md:52]: There's a filter that ignores PRs with more than 250 files. Those have been PRs that make automated script changes.

Is there a landing page to send readers to? I'll add that one by hand.

Comment (gewarren) [reply docs/whats-new/2020-09.md:52]: Yes, I think this is a good page:

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/

Comment (BillWagner) [reply docs/whats-new/2020-09.md:52]: added it.

**Pull request merged**